### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_Links_Keyboard: The keyboard focus is not visible after "default" link on pressing tab key

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#185]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG2.4.7;#Keyboard;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to "links" and tab to default link
4. Now hit tab key and check if focus is visible

### Actual:
The keyboard focus is not visible after "default" link on pressing tab key. It goes on disabled link.

### Similar issue is observed with scenarios:

1. Read only number field
2. Text Area

### Expected:
The keyboard focus should be visible after "default" link on pressing tab key. The keyboard focus should go on next actionable control and not on disabled link.

### User Impact:
Keyboard only low vision users will face difficulty in viewing the keyboard focus.

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
